Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM rsrchboy/perl-v5.16
+MAINTAINER Xinjiang Shao <shaoxinjiang@gmail.com>
+
+RUN apt-get update \
+    && apt-get install -y curl \
+    git \
+    libssl-dev \
+    libmpfr-dev \
+    nodejs \
+    npm \
+    nodejs-legacy \
+    && rm -fr /var/lib/apt/lists/*
+RUN npm install -g handlebars@1.3.0 uglifyjs
+RUN cpanm --notest --mirror http://www.cpan.org/ --mirror http://duckpan.org Dist::Zilla
+
+# Install DuckPAN 
+RUN cpanm --notest --skip-installed --mirror http://www.cpan.org/ --mirror http://duckpan.org App::DuckPAN 
```

Comment:
@soleo we've now put the DDG lib on CPAN, so you should be OK to just `cpanm App::DuckPAN` now 👍